app/angular/src/server/angular-devkit-build-webpack.ts

```diff
@@ -105,6 +105,9 @@ const buildWebpackConfigOptions = async (
     scripts: [],
     sourceMap: {},
     styles: [],
+    // Deleted in angular 12. Keep for compatibility with versions lower than angular 12
```

Comment:
I wonder, if we should change our strategy for supporting angular versions. 
For ex: 
Instead of having a single code (angular-devkit-build-webpack.ts ) which supports angular 6 to 12. 🤯 
we could cut by version. (when necessary).
This would allow to have a latest version which can contain improvements not supported by the previous ones. 
and have declinations angular6, angular 10, ... which can rely on the dependencies of angular of these versions

Reply:
@shilman @kroeder @gaetanmaisse,  what do you think?

Reply:
I think we can break compatibility with older versions of angular on Storybook major versions (e.g. in 7.0 we can break compat with Angular 9 and below, or whatever is convenient). During each release cycle (e.g. 6.x) we should do our best to preserve back compat -- not sure the best way to do this 🤷

Reply:
I do agree with you @ThibaudAV it could really simplify maintenance to have separated files for each version of Angular. 
Moreover, when we will want to remove the support of a version we will just have to remove a file 👌🏻 
Let me know if you want to do some pair programming around that :)